tests/test_notifications.py

```diff
@@ -95,7 +95,13 @@ def test_notify_user_of_landing_failure(check_celery, app, celery_worker, smtp):
     # for an observable effect of sending emails in this test because the
     # celery_worker fixture causes the test to cross threads.  We only ensure the
     # happy-path runs cleanly.
-    notify_user_of_landing_failure(Transplant(revision_order=["1"]))
+    transplant = Transplant(revision_order=["1"])
+    notify_user_of_landing_failure(
+        transplant.requester_email,
+        transplant.head_revision,
+        transplant.error,
+        transplant.request_id,
+    )
```

Comment:
Is creating the `Transplant` object is required?

Reply:
It is not required at all, indeed. I will update this in a follow up PR. This test needs to be rewritten though, as it's not really testing or doing much, as per the original comment above.